modules/exploits/unix/webapp/openmediavault_auth_cron_rce.rb

```diff
@@ -0,0 +1,384 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::Deprecated
+
+  moved_from 'exploit/multi/http/openmediavault_cmd_exec'
```

Comment:
Thanks for adding this. Please, can you also delete the `modules/exploits/multi/http/openmediavault_cmd_exec.rb` file?

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/19298/commits/c94dc8f28c2c7fee9001ea3f9315281b50e8a484.